init_krb5.sh

```diff
@@ -78,6 +78,8 @@ cat <<EOF >${WRK_DIR}/krb5.conf
   kdc_listen = 127.0.0.1:${KDC_PORT}
   kdc_tcp_listen = 127.0.0.1:${KDC_PORT}
   kdc = 127.0.0.1:${KDC_PORT}
+  kdc_ports = ${KDC_PORT}
+  kdc_tcp_ports = ""
   default_domain = KAZOOTEST.ORG
  }
 
```

Comment:
FYI typo in commit message here: "desactivate" --> "deactivate"

Doesn't matter if this will all get squashed, but looking at your clean commit history of distinct changes I'd rather not squash these as it's helpful to keep them as separate IMO